src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -722,4 +722,47 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
   <data name="Error_NoMatchingCertificate" xml:space="preserve">
     <value>The package signature did not match any of the allowed certificate fingerprints.</value>
   </data>
+  <data name="ArgumentCannotBeNullOrEmpty" xml:space="preserve">
+    <value>The argument cannot be null or empty.</value>
+  </data>
+  <data name="Error_AuthorTrustExpectedAuthorSignature" xml:space="preserve">
+    <value>Author trust was expected but package is not author signed.</value>
```

Comment:
Can this message be reworked? It doesn't flow as well as the rest.

Reply:
What about: `The packages is not author signed.`